### PR TITLE
[14.0] base_delivery_carrier_label: add check when no carrier_id assigned

### PR DESCRIPTION
**Pull request by sonhd91**
Add check when no carrier assigned on picking
Fix `import mock `
Merge with [PR#458](https://github.com/OCA/delivery-carrier/pull/458) to improve and fix conflict
* Fix `_set_default_package` from `base_delivery_carrier_label`
* Fix `delivery_carrier_pricelist` module test
* Fix  `delivery_package_fee` module test
* Add `delivery_type` check in `delivery_postlogistics`
* Add `delivery_type` check in `delivery_multi_destination`

### CODE REVIEW

**Review by leemannd — APPROVED** [review]

File: delivery_multi_destination/models/delivery_carrier.py [hunk 1]
```diff
@@ -77,33 +77,34 @@ def send_shipping(self, pickings):
...
-                        except Exception:
-                            pass
-                if not picking_res:
-                    raise ValidationError(_("There is no matching delivery rule."))
-                res += picking_res
-            return res
+            try:
+                picking_res = super().send_shipping(picking)
+            except Exception:
+                pass
```
Comment (santostelmo) [inline delivery_multi_destination/models/delivery_carrier.py]: A silent exception ?  Can we log that?

**Review by santostelmo — APPROVED** [review]
LG. just a minor comment

File: delivery_multi_destination/models/delivery_carrier.py (hunk 1 above)
Comment (sebalix) [reply delivery_multi_destination/models/delivery_carrier.py]: It was already written like this in the previous code but indeed, a error or warning log is welcome.

Comment (sonhd91) [reply delivery_multi_destination/models/delivery_carrier.py]: Thanks for remind, working on it

Comment (hailangvn) [reply delivery_multi_destination/models/delivery_carrier.py]: Good day @santostelmo, @sebalix, the PR has been updated.

**Review by sebalix — APPROVED** [review]

**Pull request merged**